CONTRIBUTING.md

```diff
@@ -47,3 +47,43 @@ To add a new locale (`fr` in this example, use only ISO-639-1):
 * Upload to PyPI `twine upload dist/youtube2zim-2.0.0*`.
 * Commit your CHANGELOG + version bump changes
 * Tag version on git `git tag -a v2.0.0`
+
+## developing the ZIM UI in Vue.JS
+
+Sometimes you need to alter something in the ZIM UI in Vue.JS but for this to work, you need assets which are generated by the scraper (e.g. channel.json, ...).
+
+To simplify this, it is possible to:
+- run the scraper (with original code base or your modified one)
+- extract assets from generated files and place them in a directory where ZIM UI will find them
+- iterate on ZIM UI code
+
+To achieve this, first build the Docker image based on current code base.
+
+```
+docker build -t local-youtube2zim .
+```
+
+Scrape a channel (here we use the [openZIM_testing](https://www.youtube.com/channel/UC8elThf5TGMpQfQc_VE917Q) channel, but you could use any other one of interest for your UI developments).
+
+```
+docker run --rm -it -v "$PWD/output":/output local-youtube2zim youtube2zim --api-key <YOUR-API-KEY> --type channel --id "UC8elThf5TGMpQfQc_VE917Q" --name "openZIM_testing" --zim-file "openZIM_testing"
+```
+
+Extract interesting ZIM content and move it to `public` folder.
+
+```
+find zimui/public/ -mindepth 1 -maxdepth 1 ! -name ".gitignore" -delete
```

Comment:
This is not working since we have content in subfolders. Since here we have nothing in `public` by default, we can just get rid of the `-maxdepth 1` so that we also explore subfolders to delete everything.

Reply:
Updated in [fad5616](https://github.com/openzim/youtube/pull/225/commits/fad56166741cedd620ddead200d9931fda3dd133)